src/routes/delegates/entities/schemas/__tests__/delete-safe-delegate.dto.schema.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { deleteSafeDelegateDtoBuilder } from '@/routes/delegates/entities/__tests__/delete-safe-delegate.dto.builder';
+import { DeleteSafeDelegateDto } from '@/routes/delegates/entities/delete-safe-delegate.dto.entity';
+import { DeleteSafeDelegateDtoSchema } from '@/routes/delegates/entities/schemas/delete-safe-delegate.dto.schema';
+import { faker } from '@faker-js/faker';
+import { getAddress } from 'viem';
+
+describe('DeleteSafeDelegateDtoSchema', () => {
+  it('should validate a DeleteSafeDelegateDto', () => {
+    const deleteSafeDelegateDto = deleteSafeDelegateDtoBuilder().build();
+
+    const result = DeleteSafeDelegateDtoSchema.safeParse(deleteSafeDelegateDto);
+
+    expect(result.success).toBe(true);
+  });
+
+  it.each(['delegate' as const, 'safe' as const])(
```

Comment:
nitpick: `it.each<keyof DeleteSafeDelegateDto>`

Reply:
Added in 40242cfa76d1d4068b0b9ad2dfc3faf1706d4da8.